src/classBased/Components/TodoContainer.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable */
+import React from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import TodosList from './TodosList';
+import Header from './Header';
+import InputTodo from './InputTodo';
```

Comment:
Please try not to disable linters, it would be better if you fix linter errors instead